macro/src/lib.rs

```diff
@@ -146,6 +161,13 @@ struct DeriveForType {
     derive: Punctuated<syn::Path, syn::Token![,]>,
 }
 
+#[derive(Debug, FromMeta)]
+struct AttributeType {
```

Comment:
```suggestion
struct AttributeForType {
```

For consistency with above :)

Reply:
:heavy_check_mark: